packages/client/src/impl/QueryApiImpl.ts

```diff
@@ -67,12 +68,9 @@ export default class QueryApiImpl implements QueryApi {
         yield row
       }
     }
-
-    flightDataStream.cancel()
   }
 
   async close(): Promise<void> {
     this.closed = true
-    this.flightClient.close()
```

Comment:
The `GrpcTransport` transport should be closed when the client is disposed:

<img width="684" alt="image" src="https://github.com/InfluxCommunity/influxdb3-js/assets/455137/c89a7b1b-7aac-416b-ae7c-b9236c8542ad">

Reply:
transport used by client is now closed inside close()